cmd/storagenode/dashboard.go

```diff
@@ -90,10 +84,20 @@ func printDashboard(data *pb.DashboardResponse, online bool) error {
 	w := tabwriter.NewWriter(color.Output, 0, 0, 1, ' ', 0)
 	fmt.Fprintf(w, "ID\t%s\n", color.YellowString(data.NodeId.String()))
 
-	if online {
-		fmt.Fprintf(w, "Status\t%s\n", color.GreenString("ONLINE"))
+	lastContacted, err := ptypes.Timestamp(data.LastPinged)
+	if err != nil {
+		lastContacted = time.Time{}
+	}
+	lastQueried, err := ptypes.Timestamp(data.LastQueried)
+	if err == nil {
+		if lastQueried.After(lastContacted) {
+			lastContacted = lastQueried
+		}
+	}
+	if lastContacted.IsZero() {
```

Comment:
Should we make this red if `(now - lastContacted > X ms) || lastContacted < now-data.GetUptime()` or

Reply:
maybe the first one? the second one will never be false since it only keeps track of last contacted in memory since uptime started. what should X be for the first one?

Reply:
Since there are no periodic pings I would probably be worried after maybe 24 hours?

Reply:
done